adapter/src/index.js

```diff
@@ -12,6 +12,7 @@ const AppAdapter = ({
     appVersion,
     url,
     apiVersion,
+    direction,
```

Comment:
do we need to set the default to 'ltr' here in case no value was provided?

Reply:
I suppose that would be good, in case something goes wrong with the env var -- which makes me wonder, should we handle 'default to LTR' here only, and leave out the default env var? 🤔 It would tidy up the env vars slightly in the default case

Reply:
yeah I would say so .. it would keep the default env vars tidier